### ISSUE TITLE
The setting parameter "Show 'add folder/channels'" doesn't work correctly in Folder list

### ISSUE THREAD

**Issue by jirkoun** [mintsoft/kodi.plugin.ytchannels#41]
Even if I have the "Show 'add folder/channels'" parameter set to 'false', the line "Add channel to <folder_name>" still appears in the Folder list.

I fixed (localy) this mismatch by being in the **ytchannels.py** module (in version 0.5.0+matrix.1) at **line 209**, instead of the original:

```
        url = build_url({'mode': 'add_channel', 'foldername': '%s'%foldername})
        li = xbmcgui.ListItem('[COLOR green]%s[/COLOR] [COLOR blue]%s[/COLOR]'%(local_string(30009),foldername))
        li.setArt({'icon':plus_img})
        xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li,isFolder=True)
```
added one line - so it's new:
```
	if show_adds !='false':
		url = build_url({'mode': 'add_channel', 'foldername': '%s'%foldername})
		li = xbmcgui.ListItem('[COLOR green]%s[/COLOR] [COLOR blue]%s[/COLOR]'%(local_string(30009),foldername))
		li.setArt({'icon':plus_img})
		xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li,isFolder=True)
```

**Comment by mintsoft** [comment]
Just confirmed this. The setting was only toggling the show/hide on the root folder. That fix above will work, I'll make that change in a sec

**Comment by mintsoft** [comment]
Fixed in 0.5.1; https://github.com/xbmc/repo-plugins/pull/4413
